spec/03-plugins/03-http-log/01-log_spec.lua

```diff
@@ -281,7 +281,13 @@ for _, strategy in helpers.each_strategy() do
       local logs = pl_file.read(test_error_log_path)
 
       for line in logs:gmatch("[^\r\n]+") do
-        assert.not_match("[error]", line, nil, true)
+        if not line:find("send(): failed request to " ..
+                         helpers.mock_upstream_ssl_host .. ":" ..
+                         helpers.mock_upstream_ssl_port .. ": timeout",
+                         0, true)
+        then
+          assert.not_match("[error]", line, nil, true)
+        end
```

Comment:
Based on the test's setup (timeout of 1 second, and posting the log to `/delay/1`, it actually sounds like we should assert that the timeout error _was_ actually hit and logged here. How about increasing the chances of hitting the timeout, and asserting the error is logged to the error log by way of `helpers.wait_until()` or even `kong stop` (to ensure logs are written to disk before reading them)?

Reply:
Yes, @hishamhm suggested this during the standup as well. I will push a commit shortly.